OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/Dockerfile

```diff
@@ -100,10 +100,8 @@ RUN mkdir /etc/envs && \
       bash -c "echo . /etc/envs/bashrc >> /etc/bashrc" && \
       bash -c "echo . /etc/envs/bashrc >> /u01/oracle/.bashrc" && \
       cd /u01/oracle && \
-      jar xf /u01/$ORACLT_PKG1 && \
-      jar xf /u01/$ORACLT_PKG2 && \
-      chmod -R +x instantclient_* && \
-      mv /u01/oracle/instantclient* /u01/oracle/instantclient && \
+      rpm -i --nodeps /u01/$ORACLT_PKG1 /u01/$ORACLT_PKG2 && \
+      mkdir /u01/oracle/instantclient && \
```

Comment:
No need to still create this directory. The previous command would install the Instant Client for you.